tests/shared_no_jsdom.js

```diff
@@ -7,7 +7,7 @@ exports.test = function(context) {
     before(function() {
       global.document = {
         get cookie() {
-          return mockStorage;
+          return mockStorage.map(cookie => cookie.split(';')[0]).join(';');
```

Comment:
I think it was a little confusing that `document.cookie` returned an array instead of a string (as the original API does)